### DISCUSSION TITLE
Are there any Chinese communication groups (中文交流群) about Vuetify?

### DISCUSSION THREAD

**Discussion by carvenl** [vuetifyjs/vuetify#18929] [Ideas]
有Vuetify的中文交流群吗？

**Comment by carvenl** [comment]
If not, I have created a new QQ group where we can communicate together.
如果没有的话，我建了一个qq群，群号668624526
有问题可以一起交流

**Reply by yuwu9145** [reply]
Discord有中文社区

**Reply by EpsilonYi** [reply]
分享一下邀请链接啊

**Reply by itancc** [reply]
https://discord.gg/vuetify
